bindings/wasm/examples-account/src/create_vp.ts

```diff
@@ -101,16 +169,19 @@ async function createVP(clientConfig) {
     const resolver = await Resolver
         .builder()
         .clientConfig({
-            network: clientConfig.network
+            network: Network.mainnet()
         })
```

Comment:
Remove the `clientConfig`, it defaults to the `mainnet` if omitted. Unless we configure the network for the accounts too.

Reply:
We have a problem with async constructors in Javascript so we just use a blocking call internally (which is not ideal), the async builder is correct. I made the `Resolver` use a builder pattern due to the difficulties of passing around clients, making duck-typed parameter difficult/impossible, if I recall correctly.